conda/resolve.py

```diff
@@ -961,41 +977,32 @@ def environment_is_consistent(self, installed):
         solution = C.sat(constraints)
         return bool(solution)
 
-    def get_conflicting_specs(self, specs):
+    def get_conflicting_specs(self, specs, original_specs=None):
         if not specs:
             return ()
-        reduced_index = self.get_reduced_index(specs)
-
-        # Check if satisfiable
-        def mysat(specs, add_if=False):
-            constraints = r2.generate_spec_constraints(C, specs)
-            return C.sat(constraints, add_if)
 
+        reduced_index = self.get_reduced_index(tuple(specs), sort_by_exactness=False)
         r2 = Resolve(reduced_index, True, channels=self.channels)
-        C = r2.gen_clauses()
-        solution = mysat(specs, True)
-        if solution:
+        scp = context.channel_priority == ChannelPriority.STRICT
+        unsat_specs = {s for s in specs if not r2.find_matches_with_strict(s, scp)}
+        if not unsat_specs:
             return ()
-        else:
```

Comment:
see https://github.com/conda/conda/pull/8741/files#r291598332 for why SAT is removed here.